File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x = 0;
        for (String s : operations) {
            switch (s) {
                case "--X":
                case "X--":
                    x--;
                    break;
                case "X++":
                case "++X":
                    x++;
                    break;
            }
        }
        return x;
    }
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations)
        {
            int x = 0;
            foreach (string s in operations)
            {
                if (s == "--X" || s == "X--")
                {
                    x -= 1;
                }
                else
                {
                    x += 1;
                }
            }
            return x;
        }
}